charts/aws-pca-issuer/templates/deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default ( printf "v%s" .Chart.AppVersion )}}"
```

Comment:
What do you think about changing the default value to latest? Is there a use case I'm missing here?

Reply:
I would not recommend not to use the latest tag as this can lead to drift between the used version inside the docker image and applied infrastructure context ( helm chart ).   

E.g. an user deploys this helm chart on staging and runs it using default values. With your proposal the user will get the newest image.  The user tests it and wants to promote this to prod, but in the meantime a new docker image version is released. Therefor the user has now two different application versions in prod and staging with the old helm chart templates running production.

Reply:
This makes sense to me, merging now